plugins/modules/nautobot_server.py

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2021, Network to Code <opensource@networktocode.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+---
+module: nautobot_server
+short_description: Manages Nautobot Server application.
+description:
+    - Manages Nautobot Server using the C(nautobot-server) application frontend to C(django-admin). With the
+      C(virtualenv) parameter, all management commands will be executed by the given C(virtualenv) installation.
+requirements:
+  - nautobot
+author:
+  - Network to Code (@networktocode)
+options:
+  command:
+    description:
+      - The name of the Nautobot management command to run. Some built in commands are C(collectstatic),
+        C(flush), C(loaddata), C(migrate), C(test), and C(validate).
+      - Other commands can be entered, but will fail if they're unknown to Nautobot. Other commands that may
+        prompt for user input should be run with the C(--noinput) flag.
+      - The module will perform some basic parameter validation (when applicable) to the commands C(collectstatic),
+        C(createcachetable), C(flush), C(loaddata), C(migrate), C(test), and C(validate).
+    type: str
```

Comment:
I kept it open because I jus tried few `commands`, but the command can work with "any" one (depending on other parameters).
So the point here is, do we want to provide only well-tested options leaving the non-tested unusable or make it a bit loose?

Reply:
leaving it open